pages/_app.tsx

```diff
@@ -68,6 +68,8 @@ function App({ Component, pageProps }) {
               });
            `}
       </Script>
+      {/* OptinMonster */}
+      <Script src="https://assets.nypl.org/js/advocacy.js" />
```

Comment:
Can you add `strategy="afterInteractive"` for minor optimization?

Reply:
I know you're copying code over from DFE and the ticket says it's optinmonster, but it doesn't _look_ like a typical optinmonster url. NBD but just for documentation purposes when others are reading this and confused about why an internal url is commented with optinmonster.